### ISSUE TITLE
Lots of false positives of vertical_parameter_alignment after switching from spaces to tabs

### ISSUE THREAD

**Issue by jbrennan** [realm/SwiftLint#2547]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

Our project was previously using spaces for indentation, and has now been converted to tabs. Unfortunately, I’m seeing hundreds of of the following violations:

```
Vertical Parameter Alignment Violation: Function parameters should be aligned vertically if they're in multiple lines in a declaration. (vertical_parameter_alignment)
```

Code sample below, but in Xcode, things look pretty lined up to me (in the screenshot I’ve turned on invisible characters)

![screen shot 2019-01-09 at 11 37 17 am](https://user-images.githubusercontent.com/131923/50913763-11f16180-1403-11e9-98c9-ac000acaac09.png)

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)?

`0.29.3`

* Installation method used (Homebrew, CocoaPods, building from source, etc)?

Cocoapods

* Paste your configuration file:

```yml
disabled_rules:
  - file_length
  - type_body_length
  - cyclomatic_complexity
  - nesting

opt_in_rules:
  - empty_count
  - force_unwrapping
  - overridden_super_call
  - redundant_nil_coalescing
  - private_outlet
  - operator_usage_whitespace
  - first_where
  - number_separator
  - prohibited_super_call
  - fatal_error_message
  - sorted_imports
  - closure_spacing
  - object_literal
  - attributes
  - toggle_bool
  - collection_alignment

custom_rules:
    rx_variable_usage:
      name: "`RxSwift.Variable` Usage"
      regex: "(Variable)"
      message: "Please use `RxCocoa.BehaviorRelay` instead."
      severity: error

    capital_id:
      name: "ID capitalization"
      regex: 'Id[^a-z]'
      message: "Prefer `ID` to `Id`"
      severity: warning

    localized_time_format:
      name: "Localized Time Format"
      regex: '\"h:mm a\"'
      message: "Please use `DateFormatter.localizedDateFormat(format: 'j:mm a')` instead."
      severity: warning

    year_formatting:
      name: "Year Formatting"
      regex: 'YY'
      message: "Please use `yy` (or, the appropriate number of digits) instead."
      severity: warning

included:
  - App
  - AppFunctionalTests
  - AppTests
  - NotificationService

excluded:
  - Pods

line_length: 200

function_body_length:
  warning: 50

identifier_name:
  min_length:
    warning: 1
  excluded:
    - id
    - x
    - y

warning_threshold: 1
indentation: tabs
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
  If so, paste their relative paths and respective contents.

no

* Which Xcode version are you using (check `xcode-select -p`)?

Version 10.1 (10B61)

* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
// This triggers a violation:
	init(apiID: String,
		 showID: String,
		 status: Status,
		 authUserReservationID: String?,
		 airTime: Date,
		 airEndTime: Date) {
```

**Comment by marcelofabri** [comment]
Duplicate of #1573
